map:
  path: "/home/tulsi/rp_project/src/xy_simulator/maps/map.png"
  width: 10.0
  height: 8.0
  resolution: 0.5
   

robots:
  - name: "Bot1"
    frame_id: "map"
    initial_pose:
      x: 7.0
      y: -11.0
      theta: -1.57
    dimensions:
      wheel_r: 3
    color:
      r: 0.0
      g: 1.0
      b: 0.0
      a: 1.0
    lidar:
      num_beams: 200
      min_range: 0.1
      max_range: 60.0
    max_vel: 1

  - name: "Bot2"
    frame_id: "map"
    initial_pose:
      x: -3.0
      y: -21.0
      theta: 3.142
    dimensions:
      wheel_r: 4
    color:
      r: 0.0
      g: 0.0
      b: 1.0
      a: 1.0
    lidar:
      num_beams: 200
      min_range: 0.1
      max_range: 60.0
    max_vel: 1
    
  - name: "Bot3"
    frame_id: "map"
    initial_pose:
      x: -22.0
      y: -21.0
      theta: 3.142
    dimensions:
      wheel_r: 4
    color:
      r: 0.0
      g: 0.0
      b: 1.0
      a: 1.0
    lidar:
      num_beams: 200
      min_range: 0.1
      max_range: 60.0
    max_vel: 1
